ansible/deploy-all.yaml

```diff
@@ -1,35 +1,38 @@
 ---
 ############################################
+# Third-party
+############################################
+- include: infra-amqp-brokers.yaml
+  when: not parasitic
+
+- include: infra-deploy-irods-elasticsearch.yaml
+  when: not parasitic
+
+############################################
```

Comment:
Either way, I thought it was decided by CORE-7345 that we will move all `infra-*` and `support-*` playbooks and roles into our internal ansible repo.

I'm not sure about the docker-compose related playbook and roles. We may still want to publish those?